exporter/exporterhelper/queued_retry.go

```diff
@@ -74,7 +79,11 @@ func (qCfg *QueueSettings) Validate() error {
 	}
 
 	if qCfg.QueueSize <= 0 {
-		return errors.New("queue size must be positive")
+		return errors.New("queue size in batches must be positive")
+	}
+
+	if qCfg.QueueSizeBytes <= 0 {
+		return errors.New("queue size in bytes must be positive")
```

Comment:
I'd like to find a way to also error if `QueueSizeBytes` is set for the non-persistent queue, but with the current default that might be difficult.

Reply:
We could set it to 0 by default and use the `Int64Max` implicitly, since it's equal to ~8 exabytes anyway, but that would require us to treat 0 as a special value, I'm not sure if it's a good or bad idea.
Also, I think that printing a warning would be better, because the non-persistent queue does not use it anyway, but this would require passing bonus arguments to this function, so it's not worth it.